earn/src/components/boost/ImportModal.tsx

```diff
@@ -110,15 +193,16 @@ export default function ImportModal(props: ImportModalProps) {
     chainId: activeChain.id,
     enabled: enableHooks,
   });
-  const shouldWriteManager = !isFetchingManager && !!manager && manager !== ALOE_II_BOOST_MANAGER_ADDRESS;
-  const shouldMint = !isFetchingManager && !!initializationData && manager === ALOE_II_BOOST_MANAGER_ADDRESS;
+  const shouldWriteManager =
+    !isFetchingManager && !!manager && manager !== ALOE_II_BOOST_MANAGER_ADDRESS[activeChain.id];
+  const shouldMint =
+    !isFetchingManager && !!initializationData && manager === ALOE_II_BOOST_MANAGER_ADDRESS[activeChain.id];
```

Comment:
To simplify these, maybe first define another boolean like `const managerIsCorrect = manager === ALOE_II_BOOST_MANAGER_ADDRESS[activeChain.id]`.